app/models/taxonomy.rb

```diff
@@ -48,6 +48,7 @@ class Taxonomy < ActiveRecord::Base
   after_create :assign_default_templates
 
   scoped_search :on => :description, :complete_enabled => :false, :only_explicit => true
+  scoped_search :on => :id
```

Comment:
On 04/18, Ohad Levy wrote:
> > @@ -48,6 +48,7 @@ class Taxonomy < ActiveRecord::Base
> >    after_create :assign_default_templates
> >
> >    scoped_search :on => :description, :complete_enabled => :false, :only_explicit => true
> > +  scoped_search :on => :id
>
> imho this should be only_explicit with no auto completer, as ID is usually not user friendly.

I tried to do so myself before merging, but for some reason these
parameters didn't work :/

>
> ---
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/theforeman/foreman/pull/3435/files/4d3873615d4b9ab38b522b981edd1142c34eb196#r60108640

--
Daniel Lobato Garcia

@dLobatog
blog.daniellobato.me
daniellobato.me

GPG: http://keys.gnupg.net/pks/lookup?op=get&search=0x7A92D6DD38D6DE30
Keybase: https://keybase.io/elobato